data/compatible_products.yml

```diff
@@ -78,10 +78,10 @@ sets:
 
     - name: Open Liberty
       vendor: IBM Corporation
-      version: 19.0.0.6
+      version: 20.0.0.3
       image: '/images/compatible_products/ibm_open_liberty.png'
       download: 'https://openliberty.io/downloads/'
-      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/439'
+      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/855'
```

Comment:
Similar comment as above.  But, I would expect the link to be for Web Profile Certification Request:  https://github.com/eclipse-ee4j/jakartaee-platform/issues/228